metecho/api/jobs.py

```diff
@@ -207,11 +204,25 @@ def _create_org_and_run_flow(
     scratch_org.owner_sf_username = sf_username or user.sf_username
     scratch_org.owner_gh_username = user.username
     scratch_org.save()
+
+    org_config = cci.project_config.keychain.get_org(scratch_org.task.org_config_name)
+    if org_config.setup_flow:
+        flow_name = org_config.setup_flow
```

Comment:
@davisagli If the config has `setup_flow` set, I assume we should not show the Radio options for tasks on that project, since the setting will be ignored in any case?

Reply:
@jgerigmeyer huh? we still need the radio buttons to pick which org_config

Reply:
D'oh -- I was confused. 👍